packages/expo-asset/src/AssetSources.ts

```diff
@@ -80,8 +80,12 @@ export function selectAssetSource(meta: AssetMetadata): AssetSource {
     ? getManifest().bundleUrl
     : null;
   if (devServerUrl) {
-    const baseUrl = new URL(devServerUrl);
-    baseUrl.pathname = meta.httpServerLocation + suffix;
+    const baseUrl = new globalThis.URL(
```

Comment:
How would a developer (like me or a contributor) know they need to use `globalThis` to use URL? Where is `globalThis` defined?

Reply:
Oops, that was from debugging the differences with the last patch.